ui/.neutrinorc.js

```diff
@@ -49,6 +49,9 @@ module.exports = {
           '/login': {
             target: 'http://localhost:3050',
           },
+          '/logout': {
```

Comment:
This will need to get added to the k8s ingress, too.  Maybe it's easier to just use `/login/logout`?